mewarpx/tests/test_assemblies.py

```diff
@@ -157,8 +157,8 @@ def test_embedded_rectangle():
     # Check phi results against reference data                            #
     #######################################################################
 
-    phi = mwxrun.get_gathered_phi_grid()[0]
-
+    phi = mwxrun.get_gathered_phi_grid(include_ghosts=False)
+    # np.save('embedded_rectangle_phi.npy', phi)
```

Comment:
What's this commented line doing here?

Reply:
It is there in case we want to regenerate the reference data.